cram_knowrob/cram_knowrob_pick_place/src/grasping.lisp

```diff
@@ -69,8 +69,16 @@
                                  )))
 
   (<- (orientation-matters ?object-type)
-    (member ?object-type (:knife :fork :spoon :cutlery :spatula)))
-
+    (member ?object-type (:knife :fork :spoon :cutlery :spatula
+                          :fork-blue-plastic
+                          :weide-milch-small
+                          :cup-eco-orange
+                          :koelln-muesli-knusper-honig-nuss
+                          :edeka-red-bowl
+                          :spoon-blue-plastic)))
+  
+  (<- (object-type-grasp :fork-blue-plastic :human-grasp))
```

Comment:
Please define object-type-grasp and other cad-model-specific knowledge in your own package.
E.g.:

```lisp
(def-fact-group alinas-object-knowledge (obj-int:object-type-grasp)
   (<- (obj-int:object-type-grasp :fork-blue-plastic :human-grasp)))
```

Reply:
This stuff is still in the PR. Why?

Reply:
This stuff is still in the PR. Why?